schema-loader/src/test/java/schema/TableTest.java

```diff
@@ -0,0 +1,67 @@
+package schema;
+
+import static org.mockito.Mockito.when;
+
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonPrimitive;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import org.assertj.core.api.Assertions;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+
+public class TableTest {
+
+  Table table;
+
+  public static final Map<String, String> TABLE_OPTIONS = new HashMap<String, String>() {{
```

Comment:
It's never used?

Reply:
Apply in [852333d](https://github.com/scalar-labs/scalardb/pull/280/commits/852333daf4d8ccda0ed3e2bed23affcd450c007a)
I'll add some more unit tests as well.